[Docs] Add note on configuration of spark-yarn mode on docker

khalidhuseynov opened the pull request:
### What is this PR for?
This adds note to configuration to make clear about `sparkmaster` hostname defined in `/etc/hosts`


### What type of PR is it?
Improvement | Documentation

### Todos
* [x] - add note

### What is the Jira issue?
n/a

### How should this be tested?
see documentation

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

AhyoungRyu commented:
The PR title says "Add note on configuration of **spark-yarn mode** on docker", 
> **Note 2:** `sparkmaster` hostname used to run docker containers below should be defined in your `/etc/hosts`." 

but this sentence is currently added under "Spark standalone mode" section. Is it intended? If not, I think it needs to be moved to [here](https://github.com/khalidhuseynov/incubator-zeppelin/blob/9e4ad6889725efbc6ee9688247ea0bc4ccf1e0b3/docs/install/spark_cluster_mode.md#spark-on-yarn-mode) I guess :)

astroshim commented:
Could you also update the `yarn` docker file path to `$ZEPPELIN_HOME/scripts/docker/spark-cluster-managers/spark_yarn_cluster` ?

khalidhuseynov commented:
@AhyoungRyu actually I meant it for all three cases (standalone, yarn, mesos) and mistakenly added it only for standalone. So I changed it now and added small note for each case. let me know if there's a better way for that. 
@astroshim fixed docker path

AhyoungRyu commented:
@khalidhuseynov I see, then the change is looks good to me.

astroshim commented on docs/install/spark_cluster_mode.md:
````diff
@@ -78,7 +81,7 @@ You can simply set up [Spark on YARN](http://spark.apache.org/docs/latest/runnin
+You can find docker script files under `scripts/docker/spark-cluster-managers/spark_yarn_cluster`.
 
 ```
 cd $ZEPPELIN_HOME/scripts/docker/spark-cluster-managers/spark_yarn
````
This line should be `cd $ZEPPELIN_HOME/scripts/docker/spark-cluster-managers/spark_yarn_cluster` .

khalidhuseynov replied:
@astroshim thanks for review, addressed in 9648e083ceb2138d145ec7110b0511dd263cd21d

astroshim commented:
LGTM

AhyoungRyu commented:
@khalidhuseynov LGTM. Will merge if there are no more discussions.
